app/src/main/java/org/onionshare/android/tor/TorManager.kt

```diff
@@ -365,18 +232,15 @@ class TorManager @Inject constructor(
 
     private fun getBridgesFromMoat(): List<String> {
         val stateDir = app.getDir("state", MODE_PRIVATE)
-        val moat = MoatApi(executableManager.obfs4Executable, stateDir, MOAT_URL, MOAT_FRONT)
+        val obfs4Executable = TorModule.getObfs4Executable(app)
+        val moat = MoatApi(obfs4Executable, stateDir, MOAT_URL, MOAT_FRONT)
         val bridges = moat.get().let {
             // if response was empty, try it again with what we think the country should be
             if (it.isEmpty()) moat.getWithCountry(locationUtils.currentCountryIso)
             else it
         }
         return bridges.flatMap { bridge ->
-            bridge.bridgeStrings.map {
-                // add snowflake params manually, if they are missing
-                val line = if (bridge.type == "snowflake" && !it.contains("url=")) {
-                    "$it $snowflakeParams"
-                } else it
+            bridge.bridgeStrings.map { line ->
```

Comment:
Meskio confirmed that Moat is now adding all snowflake params on its owns, so luckily we can remove this workaround and don't need to keep snowflakeParams accessible for this.